Packs/Mattermost/Integrations/MattermostV2/MattermostV2.yml

```diff
@@ -48,6 +49,26 @@ configuration:
   section: Collect
   advanced: true
   required: false
+- display: Types of Notifications to Send
+  name: permitted_notifications
+  type: 16
+  section: Collect
+  options:
+  - incidentAssigned
+  - assign
+  - incidentOpened
+  - incidentChanged
+  - investigationClosed
+  - investigationDeleted
+  - incidentReminderSLA
+  - taskCompleted
+  - incidentStatusChanged
+  - failedFetchIncidents
+  - mentionNew
+  - mentionOld
+  advanced: true
+  required: false
+  additionalinfo: By default, externalFormSubmit is enabled in order to allow Ask tasks to be sent correctly.
```

Comment:
```suggestion
  additionalinfo: Notifications of type 'externalAskSubmit' or 'externalFormSubmit' are not configurable because they are required to allow Ask tasks to be sent correctly.
```